lang/cs/Org.Apache.REEF.Tests/Functional/Telemetry/TestMetricsMessage.cs

```diff
@@ -57,7 +62,9 @@ private static IConfiguration DriverConfigurations()
                 .Set(MetricsServiceConfigurationModule.CounterSinkThreshold, "5")
                 .Build();
 
-            return Configurations.Merge(c1, c2);
+            var c3 = DriverMetricsObserverConfigurationModule.ConfigurationModule.Build();
+
+            return Configurations.Merge(c1, c2, c3);
```

Comment:
We can probably have more informative variable names instead of `c1`, `c2`, and `c3`